recipes/riboseed/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "riboSeed" %}
+{% set version = "0.4.16" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "abb680a514058c2233f34a3c574727f61ed65e06b2123964b359683ccf9eb9f5" %}
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://test-files.pythonhosted.org/packages/21/59/ef9dbb631f132c312bdd1cc145395050fdd0bfbe46765280c9a157367dcd/riboSeed-0.4.16.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
This needs to be all in one line, so something like `skip: [win or py2k or py34]`
But it seems to work, so I'm wondering if you really need to skip so many things.

Reply:
Great, I will change that.